### PR TITLE
Update for FLINT 2.8.4

### PR DESCRIPTION
**Pull request by fingolfin**
CC @thofma

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 
 [compat]
 BinaryProvider = "0.4, 0.5"
-FLINT_jll = "~200.800.300"
+FLINT_jll = "~200.800.400"
```
Comment (fingolfin) [inline Project.toml:16]: Aha, of course this won't work until at least https://github.com/JuliaRegistries/General/pull/49068 is merged...

Comment (fingolfin) [reply Project.toml:16]: Actually, do we *have* to increment this at all? If we leave it as-is, it'll still try to use the latest version of FLINT_jll by default, right? And if an older version is used, that's fine, except we don't get all the nice bug fixes; but that just means people are not worse off than before... 

So we could leave this as is for now, which will allow CI to pass just fine. Then we merge this PR and release an update; and afterwards it'll be fine to merge the FLINT_jll update into the general registry. This way we'll be covered at any point in time. And if we want to "force" people to use the latest FLINT, we can later push another update for LoadFlint which increments the FLINT_jll compat. (Of course that'll only have an effect if we also increase the LoadFlint compat in other packages)

Comment (thofma) [reply Project.toml:16]: I agree that we shout keep it at `0.400` to make life a bit easier.

Comment (fingolfin) [reply Project.toml]: Done

**Review by thofma — APPROVED** [review]

**Pull request merged**